app/views/help/email_conversations.md

```diff
@@ -0,0 +1,9 @@
+<% title "Why can’t I find emails about a purchase request I’m involved in?"%>
+
+Gmail can group emails into conversations based on the subject line. If you use this function, all your emails about a single purchase request will be grouped together, with only the most recent one visible in the inbox. If you see a number in parentheses after the list of senders like `(4)`, that is how many messages are in that conversation. Open that email to see all of the earlier ones.
```

Comment:
Can we indicate that the default setting of 'on' for conversation view is the recommended setup?